tests/test_memnn.py

```diff
@@ -5,42 +5,20 @@
 # LICENSE file in the root directory of this source tree.
 
 import unittest
-import io
-import contextlib
-import tempfile
-import os
-import shutil
-
-from parlai.scripts.train_model import TrainLoop, setup_args
+import parlai.core.testing_utils as testing_utils
 
 BATCH_SIZE = 1
 NUM_EPOCHS = 3
 LR = 1
 
 
-def _mock_train(**args):
-    outdir = tempfile.mkdtemp()
-    parser = setup_args()
-    parser.set_defaults(
-        model_file=os.path.join(outdir, "model"),
-        **args,
-    )
-    stdout = io.StringIO()
-    with contextlib.redirect_stdout(stdout):
-        tl = TrainLoop(parser.parse_args(print_args=False))
-        valid, test = tl.train()
-
-    shutil.rmtree(outdir)
-    return stdout.getvalue(), valid, test
-
-
 class TestMemnn(unittest.TestCase):
     """Checks that seq2seq can learn some very basic tasks."""
 
     def test_labelcands_nomemnn(self):
         """This test uses a single-turn task, so doesn't test memories."""
 
-        stdout, valid, test = _mock_train(
+        stdout, valid, test = testing_utils.train_model(dict(
             task='integration_tests:CandidateTeacher',
```

Comment:
nit: i notice in later files, you substitute `CandidateTeacher` for `candidate`, maybe do so here to keep consistent style?